src/guides/v2.3/migration/migration-migrate-settings.md

```diff
@@ -16,12 +16,12 @@ According to our data migration [order]({{ page.baseurl }}/migration/migration-m
 
 ## Before you start: routine preparations
 
-1. Log in to Magento server as [the file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
+1. Log in to Magento server (where you have M1 and M2 installed) as [the file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
```

Comment:
> (where you have M1 and M2 installed) 

I'm not sure this assumption is 100% correct. You may not have Magento 1 installed on the server where you are executing the data migration. Your Magento 2 server may only have Magento 1 database locally or the access details for the remote Magento 1 database. 

So, here we are rather talking about 

> Log in to the server with your Magento 2 instance

Reply:
Hi, @rogyar !

I agree with your notice. 

**fixed**

Could you please verify?

Thanks